src/Websites/Websites/Cmdlets/WebApps/SetAzureWebApp.cs

```diff
@@ -165,7 +169,8 @@ public override void ExecuteCmdlet()
                             Use32BitWorkerProcess =
                                 parameters.Contains("Use32BitWorkerProcess") ? (bool?)Use32BitWorkerProcess : null,
                             AutoSwapSlotName = parameters.Contains("AutoSwapSlotName") ? AutoSwapSlotName : null,
-                            NumberOfWorkers = parameters.Contains("NumberOfWorkers") ? NumberOfWorkers : WebApp.SiteConfig.NumberOfWorkers
+                            NumberOfWorkers = parameters.Contains("NumberOfWorkers") ? NumberOfWorkers : WebApp.SiteConfig.NumberOfWorkers,
+                            FtpsState = parameters.Contains("FtpsState") ? FtpsState : "AllAllowed"
```

Comment:
The default should be whatever is the existing value WebApp.SiteConfig.ftpsState - otherwise this will override the value from the current to the default "AllAllowed"